website/docs/docs/deploy/hybrid-projects.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "About Hybrid projects"
+sidebar_label: "About Hybrid projects"
+description: "Learn how to upload dbt Core artifacts into dbt Cloud to create and set up hybrid projects."
+pagination_next: "docs/deploy/hybrid-setup"
+---
+
+# About Hybrid projects <Lifecycle status='beta,enterprise'/>
+
+<IntroText>
+With Hybrid projects, your organization can adopt complementary dbt Core and dbt Cloud workflows (where some teams develop projects in dbt Core and others in dbt Cloud) and seamlessly integrate these workflows by automatically uploading dbt Core [artifacts](/reference/artifacts/dbt-artifacts) into dbt Cloud.
+</IntroText>
+
+:::tip Available in private beta
+Hybrid projects is available in private beta to [dbt Cloud Enterprise accounts](https://www.getdbt.com/pricing). To register your interest in the beta, reach out to your account representative.
+:::
+
+dbt Core users can seamlessly upload [artifacts](/reference/artifacts/dbt-artifacts) like [run results.json](/reference/artifacts/run-results-json), [manifest.json](/reference/artifacts/manifest-json), [catalog.json](/reference/artifacts/catalog-json), [sources.json](/reference/artifacts/sources-json), and so on &mdash; into dbt Cloud after executing a run in the command line interface (CLI), which helps:
+
+- Connect teams by bringing visibility so everyone can use dbt Core-generated artifacts in dbt Cloud.
+- Enhance collaboration by sharing model metadata, dependencies, and run history across teams using dbt Cloud.
```

Comment:
I didn't think of it this way but sharing run history is a great point! Using hybrid projects, the teams that are using dbt-cloud will gain visibility into core executions (if they have the right access).